docs/gas-schedule-proposal.md

```diff
@@ -0,0 +1,223 @@
+|  | CODESIZE | 2 | 1 |
+|  | CODECOPY | 2 | 3 |
+|  | GASPRICE | 2 | 1 |
+|  | RETURNDATASIZE | 2 | 1 |
+| 3E | RETURNDATACOPY | static_gas = 3 <br/> dynamic_gas = 3 * minimum_word_size + memory_expansion_cost | static_gas = 8 <br/> dynamic_gas = 3 * minimum_word_size + memory_expansion_cost |
+|  | COINBASE | 2 | 1 |
+|  | TIMESTAMP | 2 | 1 |
+|  | NUMBER | 2 | 1 |
+|  | DIFFICULTY | 2 | 1 |
+|  | GASLIMIT | 2 | 1 |
+|  | CHAINID | 2 | 1 |
+|  | SELFBALANCE | 5 | 2 |
+|  | POP | 2 | 1 |
+|  | MLOAD | 3 | 2 |
+| 52 | MSTORE | static_gas = 3 <br/> dynamic_gas = memory_expansion_cost | static_gas = 2 <br/> dynamic_gas = memory_expansion_cost |
+| 53 | MSTORE8 | static_gas = 3 <br/> dynamic_gas = memory_expansion_cost | static_gas = 2 <br/> dynamic_gas = memory_expansion_cost |
+|  | JUMP | 8 | 1 |
+|  | JUMPI | 10 | 2 |
+|  | PC | 2 | 1 |
+|  | MSIZE | 2 | 1 |
+|  | GAS | 2 | 1 |
+|  | JUMPDEST | 1 | 1 |
+|  | PUSH1 | 3 | 1 |
+|  | PUSH2 | 3 | 1 |
+|  | PUSH3 | 3 | 1 |
+|  | PUSH4 | 3 | 1 |
+|  | PUSH5 | 3 | 1 |
+|  | PUSH6 | 3 | 1 |
+|  | PUSH7 | 3 | 1 |
+|  | PUSH8 | 3 | 1 |
+|  | PUSH9 | 3 | 1 |
+|  | PUSH10 | 3 | 1 |
+|  | PUSH11 | 3 | 1 |
+|  | PUSH12 | 3 | 1 |
+|  | PUSH13 | 3 | 1 |
+|  | PUSH14 | 3 | 1 |
+|  | PUSH15 | 3 | 1 |
+|  | PUSH16 | 3 | 1 |
+|  | PUSH17 | 3 | 1 |
+|  | PUSH18 | 3 | 1 |
+|  | PUSH19 | 3 | 1 |
+|  | PUSH20 | 3 | 1 |
+|  | PUSH21 | 3 | 1 |
+|  | PUSH22 | 3 | 1 |
+|  | PUSH23 | 3 | 1 |
+|  | PUSH24 | 3 | 1 |
+|  | PUSH25 | 3 | 1 |
+|  | PUSH26 | 3 | 1 |
+|  | PUSH27 | 3 | 1 |
+|  | PUSH28 | 3 | 1 |
+|  | PUSH29 | 3 | 1 |
+|  | PUSH30 | 3 | 1 |
+|  | PUSH31 | 3 | 1 |
+|  | PUSH32 | 3 | 1 |
+|  | DUP1 | 3 | 1 |
+|  | DUP2 | 3 | 1 |
+|  | DUP3 | 3 | 1 |
+|  | DUP4 | 3 | 1 |
+|  | DUP5 | 3 | 1 |
+|  | DUP6 | 3 | 1 |
+|  | DUP7 | 3 | 1 |
+|  | DUP8 | 3 | 1 |
+|  | DUP9 | 3 | 1 |
+|  | DUP10 | 3 | 1 |
+|  | DUP11 | 3 | 1 |
+|  | DUP12 | 3 | 1 |
+|  | DUP13 | 3 | 1 |
+|  | DUP14 | 3 | 1 |
+|  | DUP15 | 3 | 1 |
+|  | DUP16 | 3 | 1 |
+|  | SWAP1 | 3 | 1 |
+|  | SWAP2 | 3 | 1 |
+|  | SWAP3 | 3 | 1 |
+|  | SWAP4 | 3 | 1 |
+|  | SWAP5 | 3 | 1 |
+|  | SWAP6 | 3 | 1 |
+|  | SWAP7 | 3 | 1 |
+|  | SWAP8 | 3 | 1 |
+|  | SWAP9 | 3 | 1 |
+|  | SWAP10 | 3 | 1 |
+|  | SWAP11 | 3 | 1 |
+|  | SWAP12 | 3 | 1 |
+|  | SWAP13 | 3 | 1 |
+|  | SWAP14 | 3 | 1 |
+|  | SWAP15 | 3 | 1 |
+|  | SWAP16 | 3 | 1 |
+|  | MCOPY | 6 | 2 |
+|  | MCOPY_COLD | 6 | 3 |
+|  | PUSH0 | 2 | 1 |
+|  | KECCAK256 | 36 | 56 |
+|  | LOG0 | 375 | 14 |
+|  | LOG1 | 750 | 23 |
+|  | LOG2 | 1125 | 25 |
+|  | LOG3 | 1500 | 27 |
+|  | LOG4 | 1500 | 27 |
+|  | EXTCODEHASH | 100 | 8 |
+|  | EXTCODESIZE | 100 | 7 |
+|  | EXTCODECOPY | 100 | 6 |
+|  | CREATE | 32000 | 15 |
+|  | CALL | 100 | 72 |
+|  | STATICCALL | 100 | 65 |
+|  | DELEGATECALL | 100 | 60 |
+|  | RETURN | 0 | 3 |
+|  | REVERT | 0 | 7 |
+
+
+| Precompile | Name | Current Gas | Proposed Gas |
+| ------------- | ------------- | ------------- | ------------- |
+|  | ECRECOVER | 3100 | 7064 |
```

Comment:
Why in the radical scenario there is more precompiles to be repriced than in the conservative scenario?

Reply:
The scale is different, so some of the precompiles that were previously within range now need lower cost.